src/app/components/LanguageSelect/index.tsx

```diff
@@ -61,6 +61,7 @@ export const LanguageMenu = () => {
         }}
         items={languageOptions}
         fill
+        a11yTitle="Language"
```

Comment:
Why a label is not handled by i18n?

Reply:
I have found a comment in the old PR "I think we want to keep this one international. Otherwise if someone gets into French, they can't get out"

Reply:
Yeah, we're inconsistent about it tho :shrug:

Reply:
Don says it's common to translate it